tests/firedrake_adjoint/test_dynamic_meshes.py

```diff
@@ -1,12 +1,14 @@
 import pytest
 from firedrake import *
 from firedrake_adjoint import *
+import ufl
 import numpy as np
 
 
 @pytest.mark.parametrize("mesh", [UnitSquareMesh(10,10)])
 def test_dynamic_meshes_2D(mesh):
-    S = VectorFunctionSpace(mesh, "CG", 1)
+    element = ufl.FiniteElement("Lagrange", mesh.ufl_cell(), 1, variant="equispaced")
+    S = VectorFunctionSpace(mesh, element)
```

Comment:
Alternatively, I think you could do `S=mesh.coordinates.function_space()` ?

Reply:
ah yes, that's better because it would always just work

Reply:
fixed now